src/property-filter/styles.scss

```diff
@@ -98,6 +116,17 @@ $operator-field-width: 120px;
     flex-grow: 2;
   }
 
+  &-multiselect-wrapper {
+    position: relative;
+    block-size: awsui.$size-vertical-input;
+    min-inline-size: 200px;
```

Comment:
- Does 200px need to be hardcoded or can it be related to something else?
- Note that the dropdown is 7 pixels wider than the controls above, is this OK?

![Screenshot 2024-10-21 at 10 55 03](https://github.com/user-attachments/assets/296b8f3c-04b1-4b86-ba81-787a77b56827)

Reply:
I also notice a slight difference in the color behavior of this embedded select in comparison with the other selects in this form (in the other ones, the selected option has blue outline only when the dropdown is hovered). But this might be out of scope

Reply:
The value 200px is not related to anything in particular - it is just somewhat reasonable width for the input.

It is ok that the dropdown is wider - it can be even wider depending on the options width.

Reply:
Yes, the hover is rather related to the multiselect and is out of scope. We will probably look deeper into that after the release.